gnomad_qc/example_notebooks/ancestry_classification_using_gnomad_rf.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>The v2 example will use our precomputed v2 PCA scores to assign the genetic ancestry of the v2 samples with the ONNX RF model. Using the v2 loadings will be the same process as shown for v3.1, it just needs to be used on a dataset aligned to GRCh37 instead of GRCh38, since the loadings are in GRCh37. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/broadinstitute/gnomad_qc/pull/366/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>